aspnetpatch-21/test.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+set -e
+
+latest_aspnet_package=2.1.1
+sdks=$(dotnet --list-sdks | awk '{print $1}')
+test_folder=testapp
+
+function create_project_for_sdk_and_package()
+{
+  local sdk=$1
+  local package=$2
+
+  # create folder
+  rm -rf $test_folder
+  mkdir $test_folder
+
+  # global.json file
+  cat >$test_folder/global.json <<EOF
+{
+  "sdk": {
+    "version": "$sdk"
+  }
+}
+EOF
+
+  # csproj file
+  cat >$test_folder/testapp.csproj <<EOF
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="$package" />
+  </ItemGroup>
+</Project>
+EOF
+
+  # Program.cs file
+  cat >$test_folder/Program.cs <<EOF
+namespace testapp
+{
+    public class Program
+    {
+        public static void Main(string[] args) { }
+    }
+}
+EOF
+}
+
+# Verify each sdk uses the latest ASP.NET Core App/All package
+for sdk in "$sdks"; do
+  for package in Microsoft.AspNetCore.App Microsoft.AspNetCore.All; do
+
+    # create project folder
+    create_project_for_sdk_and_package $sdk $package
+
+    # publish application
+    cd $test_folder
+    dotnet publish -o out >/dev/null
+
+    # read the package version
+    package_version=$(cat out/*.deps.json | jq -r '.targets[".NETCoreApp,Version=v2.1"]["testapp/1.0.0"].dependencies["'$package'"]')
```

Comment:
> aspnetpatch-21:  /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/aspnetpatch-21/test.sh: line 63: jq: command not found
> aspnetpatch-21:  cat: write error: Broken pipe

@tmds I don't think there is `jq` installed in the VM, do you want me to make it happen or rather replace it with something else?

Reply:
I find `jq` the best option for working with json in bash. Yes, please install it in the VM.

Reply:
Okay will be probably useful in the future for other stuff.

Reply:
not much better then jq. Is it our share VM?